src/ServiceInsight/ServiceInsight.csproj

```diff
@@ -239,7 +240,7 @@
     <Reference Include="System.Transactions" />
     <Reference Include="System.Windows.Forms" />
     <Reference Include="System.Windows.Interactivity, Version=4.5.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.SDK.Expression.Blend.1.0.0\lib\net45\System.Windows.Interactivity.dll</HintPath>
+      <HintPath>..\packages\Caliburn.Micro.3.2.0\lib\net45\System.Windows.Interactivity.dll</HintPath>
```

Comment:
Does this mean we can remove the installation of `Microsoft.SDK.Expression.Blend` that was added in #769?

Reply:
The expression blend SDK was removed automatically by nuget. I'm guessing yes.